docs/add-agent.md

```diff
@@ -183,6 +183,31 @@ Make sure you've read the [Actuated EULA](https://github.com/self-actuated/actua
 
     You can also run the Actuated Agent software as a systemd unit file for automatic restarts and to start upon boot-up.
 
+    An example of a systemd file you can use:
+
+    ```systemd
+    [Unit]
+    Description=Actuated Agent
+    [Service]
+    User=<user>
+    Type=simple
+    ExecStart=<user>/start.sh
```

Comment:
Let's specify a user and give a valid useradd command too.

Reply:
I updated the service file to run as root.